impeller/entity/inline_pass_context.cc

```diff
@@ -58,22 +58,26 @@ bool InlinePassContext::EndPass() {
   }
 
   if (command_buffer_) {
-    if (!command_buffer_->EncodeAndSubmit(pass_)) {
+    if (!pass_->EncodeCommands()) {
       VALIDATION_LOG
           << "Failed to encode and submit command buffer while ending "
              "render pass.";
       return false;
     }
   }
 
-  std::shared_ptr<Texture> target_texture =
+  const std::shared_ptr<Texture>& target_texture =
       GetPassTarget().GetRenderTarget().GetRenderTargetTexture();
   if (target_texture->GetMipCount() > 1) {
-    fml::Status mip_status = AddMipmapGeneration(context_, target_texture);
+    fml::Status mip_status =
+        AddMipmapGeneration(command_buffer_, context_, target_texture);
```

Comment:
We don't need to create a new cmd buffer here, we can use the existing one after ending the render pass above.

Reply:
Brandon had this same idea in the code review but we didn't have a compelling reason to make that change since this appeared to work.

Reply:
generally fewer is better.